tests/pivots/pivot_measure/pivot_measure_display_model.test.ts

```diff
@@ -1620,10 +1620,10 @@ describe("Measure display", () => {
     expect(getFormattedGrid(model)).toMatchObject({
       A20:"(#1) Pivot",  B20: "Alice",   C20: "",        D20: "Bob",     E20: "",        F20: "Total",    G20: "",
       A21: "",           B21: "m1",      C21: "m2",      D21: "m1",      E21: "m2",      F21: "m1",       G21: "m2",
-      A22: "February",   B22: "7.03%",   C22: "7.20%",   D22: "0.00%",   E22: "0.31%",   F22: "7.03%",    G22: "7.51%",
-      A23: "March",      B23: "39.16%",  C23: "38.75%",  D23: "19.99%",  E23: "19.93%",  F23: "59.15%",   G23: "58.68%",
-      A24: "April",      B24: "25.70%",  C24: "25.54%",  D24: "8.12%",   E24: "8.28%",   F24: "33.82%",   G24: "33.81%",
-      A25: "Total",      B25: "71.89%",  C25: "71.49%",  D25: "28.11%",  E25: "28.51%",  F25: "100.00%",  G25: "100.00%",
+      A22: "February",   B22: "7.03%",   C22: "10.64%",  D22: "0.00%",   E22: "6.25%",   F22: "7.03%",    G22: "16.89%",
```

Comment:
`m2` is really not equal to `m1 + 10%` there ... Is there something I'm missing ?

Reply:
Yes, you're probably missing the `display: "%_of_grand_total"` :wink: 
It's `m1 + 10%`, then that value is transformed/displayed according to `"%_of_grand_total"`